packages/components/src/card/src/Card.css

```diff
@@ -98,25 +103,25 @@
 }
 
 /* ASIDE | HORIZONTAL */
-.o-ui-card-horizontal > .o-ui-card-aside {
+.o-ui-card-horizontal>.o-ui-card-aside {
     grid-area: aside;
     align-content: start;
     padding: var(--o-ui-sp-6);
 }
 
-.o-ui-card-horizontal > .o-ui-card-illustration + .o-ui-card-aside {
+.o-ui-card-horizontal>.o-ui-card-illustration+.o-ui-card-aside {
     padding-left: var(--o-ui-sp-6);
 }
 
 /* ASIDE | VERTICAL */
-.o-ui-card-vertical > .o-ui-card-aside {
+.o-ui-card-vertical>.o-ui-card-aside {
     grid-column-start: 1;
     grid-column-end: 3;
     padding: var(--o-ui-sp-5);
 }
 
-.o-ui-card-vertical > .o-ui-card-illustration + .o-ui-card-aside,
-.o-ui-card-vertical > .o-ui-card-image + .o-ui-card-aside {
+.o-ui-card-vertical>.o-ui-card-illustration+.o-ui-card-aside,
+.o-ui-card-vertical>.o-ui-card-image+.o-ui-card-aside {
```

Comment:
There is a lot of css files with spaces around > and +. Should you fix them too?

Reply:
space should be there actually easier to read

Reply:
@fraincs have a look also in https://github.com/gsoft-inc/sg-orbit/pull/940/files then, space have been removed around the +